CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing to Tlog
+The Tlog project welcomes new contributors. This document will guide you through the process.
+
+## Pull Requests
+We actively welcome your pull requests. There are a few guidelines that we need
+contributors to follow so that we can try to keep the codebase consistent and clean.
+
+* Fork the main repository and clone it to your local machine.
+* Create a topic branch from master. Please avoid working directly on the ```master``` branch.
+* Make commits of logical units, and make sure to follow the CODINGSTYLE.
+* Submit the Pull Request(PR) on Github.
+
+## Issues
+We use GitHub [issues tracker](https://github.com/Scribery/tlog/issues) to track public bugs.
```

Comment:
We don't have any other bugs but "public", so it's better to remove that word lest somebody thinks we're hiding something.